fuzzing/requirements.txt

```diff
@@ -1 +1,4 @@
-absl-py>=0.9.0
+# Python requirements for the tools supporting the fuzzing rules. These are
+# installed automatically through the WORKSPACE configuration.
+
+absl-py>=0.11.0
```

Comment:
nit: >= can be risky since you never know a new version breaks stuff, but upto you, we freeze version in other places.